diff -ruN wmeyes.orig/wmeyes.c wmeyes/wmeyes.c
--- wmeyes.orig/wmeyes.c	Sat May 23 19:44:25 1998
+++ wmeyes/wmeyes.c	Sun May 24 09:28:20 1998
@@ -74,8 +74,9 @@
 void scanArgs(int argc,char *argv[]);
 unsigned long getColor(char *colorName);
 
-void do_nothing(void)
+void do_nothing(int signum)
 {
+  signal (SIGALRM, do_nothing);
 }
 
 int main(int argc, char *argv[])
@@ -261,7 +262,7 @@
 	}
 	oldrelx = relx;
 	oldrely = rely;
-	if (idletime > 480)  {
+	if (idletime > 100)  {
 		if (masknum < 10)  {
 		   masknum++;
 		}
